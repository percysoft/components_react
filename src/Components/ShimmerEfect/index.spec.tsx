import * as React from 'react';
import {  mount } from 'enzyme';
import { ShimmerEffect } from './index';

describe('Shimmer component', () => {
  describe('Render snapshot', () => {
    test('Render correctly', () => {
      const wrapper = mount(
        <ShimmerEffect/>);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});

describe('Shimmer component', () => {
  describe('Render snapshot with propertys', () => {
    test('Render correctly', () => {
      const wrapper = mount(
        <ShimmerEffect
          width= "500px"
          heigth= "40px"
        />);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
