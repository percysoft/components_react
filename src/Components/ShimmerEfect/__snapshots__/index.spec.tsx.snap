// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shimmer component Render snapshot Render correctly 1`] = `"<div><div class=\\"sc-bdVaJa dZwdru\\"></div></div>"`;

exports[`Shimmer component Render snapshot with propertys Render correctly 1`] = `"<div><div class=\\"sc-bdVaJa dZwdru\\" data-width=\\"500px\\" data-heigth=\\"40px\\"></div></div>"`;
